packages/legacy-scripting-runner/test/example-app/api-key-auth.js

```diff
@@ -40,13 +32,4 @@ module.exports = {
       },
     ],
   },
-  beforeRequest: [
-    {
-      source: beforeRequestSource,
-      args: ['request', 'z', 'bundle'],
-    },
-  ],
-  afterResponse: [
-    { source: afterResponseSource, args: ['response', 'z', 'bundle'] },
-  ],
```

Comment:
I don't understand why we are and can remove these middlewares for the `custom`, `oauth2` and `session` examples as we do have middleware for them in https://github.com/zapier/zapier-platform/blob/4bae98de1deb3aae3e9c2fa0aede140fad02fde7/packages/legacy-scripting-runner/middleware-factory.js.

Reply:
Eventually these auth definitions will be merged to the main app definition. For example, see https://github.com/zapier/zapier-platform/blob/4bae98de1deb3aae3e9c2fa0aede140fad02fde7/packages/legacy-scripting-runner/test/integration-test.js#L14-L16 and https://github.com/zapier/zapier-platform/blob/4bae98de1deb3aae3e9c2fa0aede140fad02fde7/packages/legacy-scripting-runner/test/integration-test.js#L59-L62 

so we only need to write middlewares once in the main app definition.

Reply:
Ah, ok. I wonder if developers that look at these examples may get confused and think these middleware are not needed, while this is only due to how we merge in the test app.

Reply:
> developers that look at these examples may get confused

Hopefully, that doesn't happen. Those "examples" are only for us to run tests, they're never meant to be examples for external developers.